draft-ietf-tls-tls13.md

```diff
@@ -5924,6 +5924,21 @@ server-to-client encryption keys because that would entail
 the reuse of those keys. This parallels the use of the early
 application traffic keys only in the client-to-server direction.
 
+## Static (EC)DH Shares
+
+For various reasons, some implementations may be tempted to use static
```

Comment:
Implementations probably should not be assumed to have motive, whereas implementors would.